ui_framework/components/expression/_expression.scss

```diff
@@ -0,0 +1,107 @@
+$expressionColorHighlight: #00A69B;
+
+.kuiExpression {
+
+  .kuiExpression__item {
+    display: inline-block;
+    margin-right: 10px;
+    position: relative;
+  }
+
+  .kuiExpression__button {
+    background-color: transparent;
+    padding: 5px 0px;
+    border: none;
+    border-bottom: dotted 2px $globalBorderColor;
+    font-size: $globalFontSize;
+    cursor: pointer;
+
+    .kuiExpression__buttonDescription {
+      color: $expressionColorHighlight;
+      text-transform: uppercase;
+    }
+
+    .kuiExpression__buttonValue {
+      color: $globalTextColor;
+      text-transform: lowercase;
+    }
+  }
+
+  .kuiExpression__button--active {
+    border-bottom: solid 2px $expressionColorHighlight;
+  }
+
+  .kuiExpression__popover {
+    position: absolute;
+    top: calc(100% + 15px);
+    display: flex;
+    flex-direction: column;
+    flex: 1 1 auto;
+    background-color: white;
+    border: 1px solid $globalBorderColor;
+    border-radius: 6px;
+    box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.1);
+    visibility: visible;
+    opacity: 1;
+    transform: translateY(-5px) translateZ(0);
+    transition: transform $globalAnimSpeedNormal $globalAnimSlightBounce, opacity $globalAnimSpeedNormal $globalAnimSlightBounce;
+
+    // Angular hide blocks animation.
+    &.ng-hide, &.kuiExpression__popover--hidden {
```

Comment:
Can we split these onto two lines, and rename the class using the state format? And can we adjust the comment a bit?

```scss
/**
 * 1. Angular's ng-hide uses display: none to hide things, but we want to use animation to transition the element in and out.
 */
&.ng-hide,
&.kuiExpression__popover-isHidden {
  display: block !important; /* 1 */
```